packages/admin-page/composer.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "automattic/jetpack-admin-page",
+	"description": "An admin page ui tool",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-logo": "@dev"
+	},
+	"autoload": {
+		"psr-4": {
+			"Automattic\\Jetpack\\AdminPage\\": "src/"
+		}
+	},
+	"repositories": [
```

Comment:
Hm, that's interesting. Should we have this in every package as well?

Reply:
Looking into the JSON schema at https://getcomposer.org/doc/04-schema.md#repositories, I can see the `"path"` type is not listed as supported there.

What is the best course of action then?

Reply:
`path` is a valid one, here's where it is documented: https://getcomposer.org/doc/05-repositories.md#path